pygmt/__init__.py

```diff
@@ -76,6 +76,21 @@
 _atexit.register(_end)
 
 
+def _get_gmt_version():
```

Comment:
I've added a utility function `_get_gmt_version` in the `pygmt/__init__.py` file. I wonder if it's better to define a package-level variable `__gmt_version__` instead.

Reply:
Sounds good. This would be similar to `__gdal_version__` in `rasterio`, see https://github.com/rasterio/rasterio/blob/6bf045cc5e02c9f28e78d48b97b308490207d27d/rasterio/__init__.py#L84-L86

Reply:
Sounds good!